build/Microsoft.DotNet.Cli.Test.targets

```diff
@@ -112,7 +111,7 @@
   <Target Name="SetupRestoreTestsInputs"
           DependsOnTargets="PrepareTests;" >
     <ItemGroup>
-      <RestoreTestsInputs Include="$(TestDirectory)/**/*.csproj"
+      <RestoreTestsInputs Include="$(TestDirectory)/Microsoft.DotNet.Cli.Tests.sln"
```

Comment:
You can remove the exclude. Also, I don't think we need the itemgroup anymore and can just pass this path to <DotNetRestore> as is, no?